src/contract_wrappers/tokens/__tests__/security_token_wrapper.test.ts

```diff
@@ -0,0 +1,2862 @@
+      // Stub the request
+      when(
+        mockedMethod.callAsync(
+          objectContaining(dateToBigNumber(mockedParams.start)),
+          objectContaining(dateToBigNumber(mockedParams.end)),
+        ),
+      ).thenResolve(expectedResult);
+
+      // Real call
+      const result = await target.iterateInvestors(mockedParams);
+      // Result expectation
+      expect(result).toBe(expectedResult);
+      // Verifications
+      verify(mockedContract.iterateInvestors).once();
+      verify(
+        mockedMethod.callAsync(
+          objectContaining(dateToBigNumber(mockedParams.start)),
+          objectContaining(dateToBigNumber(mockedParams.end)),
+        ),
+      ).once();
+    });
+  });
+
+  describe('getInvestorCount', () => {
+    test('should call to getInvestorCount', async () => {
+      const expectedResult = new BigNumber(0);
+      // Mocked method
+      const mockedMethod = mock(MockedCallMethod);
+      // Stub the method
+      when(mockedContract.getInvestorCount).thenReturn(instance(mockedMethod));
+      // Stub the request
+      when(mockedMethod.callAsync()).thenResolve(expectedResult);
+
+      // Real call
+      const result = await target.getInvestorCount();
+      // Result expectation
+      expect(result).toBe(expectedResult);
+      // Verifications
+      verify(mockedContract.getInvestorCount).once();
+      verify(mockedMethod.callAsync()).once();
+    });
+  });
+
+  describe('checkPermission', () => {
+    test.todo('should fail as modudelegateAddressle is not an Eth address');
+    test.todo('should fail as moduleAddress is not an Eth address');
+
+    test('should call to checkPermission', async () => {
+      const expectedResult = true;
+      const mockedParams = {
+        delegateAddress: '0x1111111111111111111111111111111111111111',
+        moduleAddress: '0x1111111111111111111111111111111111111111',
+        permission: 'string',
+      };
+      // Mocked method
+      const mockedMethod = mock(MockedCallMethod);
+      // Stub the method
+      when(mockedContract.checkPermission).thenReturn(instance(mockedMethod));
+      // Stub the request
+      when(
+        mockedMethod.callAsync(
+          mockedParams.delegateAddress,
+          mockedParams.moduleAddress,
+          stringToBytes32(mockedParams.permission),
+        ),
+      ).thenResolve(expectedResult);
+
+      // Real call
+      const result = await target.checkPermission(mockedParams);
+      // Result expectation
+      expect(result).toBe(expectedResult);
+      // Verifications
+      verify(mockedContract.checkPermission).once();
+      verify(
+        mockedMethod.callAsync(
+          mockedParams.delegateAddress,
+          mockedParams.moduleAddress,
+          stringToBytes32(mockedParams.permission),
+        ),
+      ).once();
+    });
+  });
+
+  describe('getCheckpointTimes', () => {
+    test('should call to getCheckpointTimes', async () => {
+      const expectedResult = new BigNumber(0);
```

Comment:
getCheckpointTimes currently returns an array of BigNumbers. Same case than `getModulesByName`, it is better to fix it now than get an error in the future.